pkg/serviceimport/map.go

```diff
@@ -54,7 +54,7 @@ func (m *Map) selectIP(queue []clusterInfo, counter *uint64, name, namespace str
 }
 
 func (m *Map) GetIP(namespace, name, cluster, localCluster string, checkCluster func(string) bool,
-	checkEndpoint func(string, string, string) bool) (string, bool) {
+	checkEndpoint func(string, string, string) bool) (ip string, found, isLocal bool) {
```

Comment:
The named return vars are not used. Are you naming them just for documentation?

Reply:
Yes, for documentation.